### ISSUE TITLE
Can't get redirects to work

### ISSUE THREAD

**Issue by rstacruz** [firebase/superstatic#201]
I want to make `http://site.com/portfolio` to redirect to `http://site.com/`. This is my superstatic config, which doesn't seem to do what I hope it does:

```js
{
  "cwd": "public",
  "cleanUrls": true,
  "function()": [
    { "source": "portfolio", "destination": "/" },
    { "source": "/portfolio", "destination": "/" },
    { "source": "portfolio", "destination": "" },
    { "source": "/portfolio", "destination": "" }
  ]
}
```

**Comment by rstacruz** [comment]
it seems like this is the correct way:

```js
{
  "cwd": "public",
  "cleanUrls": true,
  "redirects": [
    { "source": "/portfolio", "destination": "/" }
  ]
}
```

...but the docs still say it should be `function()`. Is that a typo?

> ![pasted_image_1_5_16__3_25_am](https://cloud.githubusercontent.com/assets/74385/12098256/fff19424-b35b-11e5-8993-fd80e829775b.jpg)

**Comment by mbleigh** [comment]
Whoops! Thanks for the fix in #202
